c++/CMakeLists.txt

```diff
@@ -64,7 +63,6 @@ endif()
 if(MSVC)
   # TODO(cleanup): Enable higher warning level in MSVC, but make sure to test
   #   build with that warning level and clean out false positives.
-  set(CMAKE_CXX_FLAGS "${CAPNP_LITE_FLAG} ${CMAKE_CXX_FLAGS}")
```

Comment:
It works thanks to `target_compile_definitions(kj PUBLIC ${CAPNP_LITE_FLAG})` in `src/kj/CMakeList.txt`